base/lnls-prune-artifacts.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -eu
+
+target_paths=$@
+
+
+get_used_epics_modules() {
+    targets=$@
+
+    executables=$(find $targets -type f -executable -print)
+    libs="$(ldd $executables 2>/dev/null)"
```

Comment:
`ldd` on musl doesn't check all arguments, it simply ignores all but the first one.

We should either explicitly loop or add a comment about depending on this behavior.